cetmix_tower_server/tests/test_plan.py

```diff
@@ -11,6 +11,14 @@
 class TestTowerPlan(TestTowerCommon):
     def setUp(self, *args, **kwargs):
         super().setUp(*args, **kwargs)
+        # Tags
+        self.tag_test_staging = self.env["cx.tower.tag"].create(
+            {"name": "Test Staging"}
+        )
+        self.tag_test_production = self.env["cx.tower.tag"].create(
```

Comment:
You have already created these tags in the `common.py`, why are you creating them again?

Reply:
Corrected.